frontend/src/components/App/Settings/NumRowsInput.tsx

```diff
@@ -157,6 +157,7 @@ export default function NumRowsInput(props: { defaultValue: number[] }) {
           renderValue={value => `${value}`}
           size="small"
           variant="outlined"
+          {...(tableID && { labelId: tableID, 'aria-labelledby': tableID })}
```

Comment:
aria-labelledby is not required, https://mui.com/material-ui/react-select/#accessibility